test/built-ins/Temporal/PlainDateTime/prototype/until/returns-days.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.until
+description: Return days by default
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const feb20 = new Temporal.PlainDateTime(2020, 2, 1, 0, 0);
+const feb21 = new Temporal.PlainDateTime(2021, 2, 1, 0, 0);
+
+TemporalHelpers.assertDuration(
```

Comment:
There's some overlap between this file and the previous one.

Reply:
Hmm, the previous test doesn't explicitly test `largestUnit` = `auto`, so to my eyes this seems at least superficially different. Perhaps I'm missing something?

Reply:
I'm guessing "some overlap" doesn't mean "completely superfluous" - I think it's fine to leave this as is, and I'll take another look at these tests next week.